data/ui_dashboards/ontologies/Chart types.csv

```diff
@@ -0,0 +1,10 @@
+order,name,label,tags,parent,codesystem,code,ontologyTermURI,definition
+,data highlight,,,,,,,
```

Comment:
nice

Reply:
Unfortunately, there isn't an ontology or set of terms for visualisation.